string graph_name
bool name_match_required
string[] targets
string[] umrf_json_strings
